spec/feature/users_show_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'User show page', type: :feature do
+  let!(:user) { User.create(name: 'Sarkodie', photo: '/path/to/photo1.jpg', post_counter: 10) }
+
+  before do
+    5.times do
+      Post.create(
+        title: 'Post title',
+        text: 'Post text',
+        author_id: user.id
+      )
+    end
+    visit user_path(user)
+  end
+
+  it 'User profile information' do
+    expect(page).to have_css("img[src='#{user.photo}'][alt='user photo']")
+    expect(page).to have_content(user.name)
+    expect(page).to have_content("Number of posts: #{user.post_counter}")
+    expect(page).to have_content(user.bio)
+  end
+
+  it 'Displaying first 3 posts' do
+    user.posts.order(created_at: :desc).limit(3).each do |post|
+      expect(page).to have_content(post.title)
+      expect(page).to have_content(post.text)
+    end
+  end
+
+  it 'View all posts button redirects to user posts index page' do
+    click_link 'See all posts'
+    expect(current_path).to eq(user_posts_path(user))
+  end
+end
```

Comment:
- [ ] For the `Users#Show` page, you forgot to write tests that confirm the following:

- **_Tests that assert you can see a button that lets me view all of a user's posts._**
- **_Tests that redirect to the post's show page when you click on a user's post._**

Kindly write these tests. 👍 😎 

#### N.B:

Please note that the **_Tests that assert you can see a button that lets me view all of a user's posts._** is different from _**Tests that redirect to the user's post's index page when you click on a **see all posts**.**_ The former seeks to ensure that there is the **_See All Posts_** button is present on the page. 💻 

![Screenshot](https://github.com/CaptainBawa/Blog-App/assets/77106962/25d419f0-4235-40cf-8e6a-c6bf7fba607b)